contrib/profiler/run.sh

```diff
@@ -16,52 +16,86 @@
 # DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-PYTHON_VERSION=`python -V 2>&1|awk '{print $2}'|awk -F '.' '{print $1}'`
+case $1 in
+  -h|--help)
+    echo "usage: run.sh [-c <container_id>]
+    [-g <GPU index separated by ','>]
+    [-s <sample period>]
+    [-a <analyze period>]"
+    exit 0
+    ;;
+esac
+
+# Install NFS
+apt update
+apt install --assume-yes nfs-common cifs-utils sshpass wget
+# mount the server root directory to the container root directory
+mkdir --parents /mnt/data_for_profiler
+mount -t nfs4 10.151.41.14:/data/share/drbdha/data/ /mnt/data_for_profiler
```

Comment:
Why need nfs here? And the IP address is different for each cluster, do not hard code here